axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
But if the players no longer read the data then the data and these functions are unnecessary (and their coverage will disappear). So wouldn't we just delete the data and these functions in that case, unless something else is using them? And if something else is using them then a bad change will still break those things.